src/components/common/Badge/EmojiBadge.jsx

```diff
@@ -0,0 +1,36 @@
+import { cva } from 'class-variance-authority';
+import PropTypes from 'prop-types';
+import { cn } from '@/utils/style';
+
+// 이모지 뱃지 스타일 정의
+const emojiBadgeStyle = cva(
+  'inline-flex items-center justify-center rounded-full bg-black/50 text-white font-16-regular',
+  {
+    variants: {
+      size: {
+        md: 'px-3 py-2 h-[36px] text-sm',
+        lg: 'px-4 py-2 h-[40px] text-base',
+      },
+    },
+    defaultVariants: {
+      size: 'md',
+    },
+  }
+);
+
+export const EmojiBadge = ({ emoji, count, size = 'md' }) => {
```

Comment:
export가 중복되고 있습니다!
아래 default export 구문이 있어서 const 키워드 앞에 export는 제거해야할 것 같습니다.
```suggestion
const EmojiBadge = ({ emoji, count, size = 'md' }) => {
```